Entity/StockInfo.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace CoffeeBike\CollmexBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="cm_stock_info")
+ */
+class StockInfo extends CollmexObject
+{
+    /**
+     * @ORM\Column(type="string")
+     */
+    protected $type_identifier ='CMXPRD';
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="string")
+     */
+    protected $product_id;
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     */
+    protected $company_id;
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     */
+    protected $stock_available;
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="string")
+     */
+    protected $volume_unit;
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     */
+    protected $reorder_time;
+
+
+
+    public function getExtraInfo()
```

Comment:
Is getExtraInfo() and parseRemark() required here? I think it's only used by product entities.